### PR TITLE
PART 2 - refactor plugin

### PR DESCRIPTION
**Pull request by oleksandrkozlov1**
Task: AS A USER, I WANT to revise com.lgc.dsp.ui.d21 plugin code for duplicated eclipse logic, and move the eclipse logic to the eclipse code base SO THAT I won't have redundant code.

### CODE REVIEW

**Comment by ybova** [comment]
Version update is missing

**Comment by ybova** [comment]
Version update is missing

**Comment by IlyaMikhailovHB** [comment]
I think we need to make this code working even without d21 styles. So that d21 styles should be optional.

**Comment by oleksandrkozlov1** [comment]
done.

**Comment by ybova** [comment]
Version update in pom.xml is still missing.

**Comment by oleksandrkozlov1** [comment]
done,

File: bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java [hunk 1]
```diff
@@ -1,1222 +1,1295 @@
...
+				y = onBottom ? y - paddingTop - marginHeight - borderTop - (cornerSize / 4)
+						: y - paddingTop - marginHeight - tabHeight - borderTop - (cornerSize / 4);
+				width = 2 + paddingLeft + paddingRight;
+				height += paddingTop + paddingBottom;
+				height += tabHeight + (cornerSize / 4) + borderBottom + borderTop;
+			} else {
+				x = x - marginWidth - OUTER_KEYLINE - INNER_KEYLINE - sideDropWidth - (cornerSize / 2);
+				width = width + 2 * OUTER_KEYLINE + 2 * INNER_KEYLINE + 2 * marginWidth + 2 * sideDropWidth
+						+ cornerSize;
+				int tabHeight = parent.getTabHeight(); // TODO: Figure out
```
Comment (ybova) [inline bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java]: Is this TODO still relevant?

Comment (OSKozlov) [reply bundles/org.eclipse.e4.ui.workbench.renderers.swt/src/org/eclipse/e4/ui/workbench/renderers/swt/CTabRendering.java]: removed. done.

**Pull request closed without merging**